packages/shared/src/components/streak/popup/DayStreak.tsx

```diff
@@ -50,9 +52,17 @@ export function DayStreak({
         className={classNames(
           className,
           iconSizeToClassName[size],
-          streak === Streak.Freeze && 'bg-text-disabled',
+          streak === Streak.Freeze &&
```

Comment:
`const isStreakFreeze = streak === Streak.Freeze` and reuse down the line

Reply:
Added